webpack.config.js

```diff
@@ -14,6 +14,7 @@ if (process.env.SENTRY_STATIC_DIST_PATH) {
 }
 
 var IS_PRODUCTION = process.env.NODE_ENV === 'production';
+var REFRESH = process.env.REFRESH === 'true';
```

Comment:
I find it a little weird to make the value of this `'true'` (since in shell the string `'false'` evaluates as truthy), but hey wfm.